js-api-spec/function.test.ts

```diff
@@ -77,6 +82,129 @@ it('passes a default argument value', () => {
   expect(fn).toHaveBeenCalled();
 });
 
+describe('simplifies', () => {
```

Comment:
Nit: Since this is so calculation-specific, I'd expect it to be in the calculation test suite.